Esignet/scripts/Esignet_Test_Script.jmx

```diff
@@ -1709,6 +1923,15 @@ vars.put(&quot;publicKey&quot;, publicKey);
             <stringProp name="RegexExtractor.match_number">1</stringProp>
           </RegexExtractor>
           <hashTree/>
+          <RegexExtractor guiclass="RegexExtractorGui" testclass="RegexExtractor" testname="Extract Captcha" enabled="true">
```

Comment:
For capcha, what are we doing ?

Reply:
We are getting the captcha value which is generated from the preparation thread group response. And we are storing the value in a file and using the same in the execution.

Reply:
Are you sure this works ? Was the captcha enabled when the script were run ?

Reply:
Captcha is disabled. Have updated the endpoint will captcha field as empty.